Packs/EclecticIQ/pack_metadata.json

```diff
@@ -2,11 +2,11 @@
     "name": "EclecticIQ Platform",
     "description": "Threat Intelligence Platform that connects and interprets intelligence data from open sources, commercial suppliers and industry partnerships.",
     "support": "xsoar",
-    "currentVersion": "1.0.5",
+    "currentVersion": "2.0.0",
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
```

Comment:
change URL and email to relevant for your company that can be reachable for users to reach out in case they need

Reply:
Updated as per the comment